tests/unit/40_nginx-vhosts_3.bats

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bats
+
+load test_helper
+source "$PLUGIN_CORE_AVAILABLE_PATH/config/functions"
+
+setup() {
+  global_setup
+  [[ -f "$DOKKU_ROOT/VHOST" ]] && cp -fp "$DOKKU_ROOT/VHOST" "$DOKKU_ROOT/VHOST.bak"
+  [[ -f "$DOKKU_ROOT/HOSTNAME" ]] && cp -fp "$DOKKU_ROOT/HOSTNAME" "$DOKKU_ROOT/HOSTNAME.bak"
+  create_app
+  docker build "${BATS_TEST_DIRNAME}/../../tests/apps/gogrpc" -t ${TEST_APP}-docker-image
+}
+
+teardown() {
+  destroy_app 0 $TEST_APP
+  [[ -f "$DOKKU_ROOT/VHOST.bak" ]] && mv "$DOKKU_ROOT/VHOST.bak" "$DOKKU_ROOT/VHOST" && chown dokku:dokku "$DOKKU_ROOT/VHOST"
+  [[ -f "$DOKKU_ROOT/HOSTNAME.bak" ]] && mv "$DOKKU_ROOT/HOSTNAME.bak" "$DOKKU_ROOT/HOSTNAME" && chown dokku:dokku "$DOKKU_ROOT/HOSTNAME"
+  global_teardown
+}
+
+@test "(nginx-vhosts) grpc endpoint" {
+  deploy_app gogrpc
+  dokku proxy:ports-add "$TEST_APP" "grpc:80:50051"
```

Comment:
Can you add a test where we use `8080` instead of port `80` for the host as well?